src/overlays/actors/ovl_En_Hg/z_en_hg.c

```diff
@@ -15,21 +15,36 @@ void EnHg_Destroy(Actor* thisx, PlayState* play);
 void EnHg_Update(Actor* thisx, PlayState* play);
 void EnHg_Draw(Actor* thisx, PlayState* play);
 
-void func_80BCF354(EnHg* this);
-void func_80BCF398(EnHg* this, PlayState* play);
-void func_80BCF468(EnHg* this);
-void func_80BCF468(EnHg* this);
-void func_80BCF4AC(EnHg* this, PlayState* play);
-void func_80BCF5F0(EnHg* this);
-void func_80BCF634(EnHg* this, PlayState* play);
-void func_80BCF68C(EnHg* this);
-void func_80BCF6D0(EnHg* this, PlayState* play);
-void func_80BCF8A0(EnHg* this, PlayState* play);
-void func_80BCF93C(EnHg* this);
-void func_80BCF95C(EnHg* this, PlayState* play);
+void EnHg_SetupIdle(EnHg* this);
+void EnHg_Idle(EnHg* this, PlayState* play);
+void EnHg_SetupMoveTowardsPlayer(EnHg* this);
+void EnHg_MoveTowardsPlayer(EnHg* this, PlayState* play);
+void EnHg_SetupDetectPlayerDistanceFromHomePos(EnHg* this);
+void EnHg_DetectPlayerDistanceFromHomePos(EnHg* this, PlayState* play);
+void EnHg_SetupReactToHit(EnHg* this);
+void EnHg_ReactToHit(EnHg* this, PlayState* play);
+void EnHg_PlayCutscene(EnHg* this, PlayState* play);
+void EnHg_SetupCsAction(EnHg* this);
+void EnHg_HandleCsAction(EnHg* this, PlayState* play);
 s32 EnHg_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx);
 void EnHg_PostLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx);
 
+typedef enum {
+    /* 0x00 */ HG_ANIM_NO_ACTION,
+    /* 0x03 */ HG_ANIM_LEAN_FORWARD = 3,
+    /* 0x04 */ HG_ANIM_REACH_FORWARD,
+    /* 0x05 */ HG_ANIM_CURL_UP,
+    /* 0x06 */ HG_ANIM_CROUCHED_PANIC,
+    /* 0x07 */ HG_ANIM_PANIC,
+} HgCsAnimation;
```

Comment:
```suggestion
    /* 0x00 */ HG_ANIM_NO_ACTION,
    /* 0x03 */ HG_ANIM_LEAN_FORWARD = 3,
    /* 0x04 */ HG_ANIM_REACH_FORWARD,
    /* 0x05 */ HG_ANIM_CURL_UP,
    /* 0x06 */ HG_ANIM_CROUCHED_PANIC,
    /* 0x07 */ HG_ANIM_PANIC,
} HgAnimation;
```
Doesn't look like it is tied to cutscenes, just the animations in general. Also since you have the full array, could you fill out the rest of the enum even if unused.